README.md

```diff
@@ -153,7 +153,7 @@ Note: The C tests have to be built by ``make tests``)!
 
 ### Run only Python unittests (from your checkout dir):
 
-    PYTHONPATH=`readlink -f ./build/src/python/` nosetests -s tests/python/tests/
+    PYTHONPATH=`readlink -f ./build/src/python/` python3 -m unittest discover -bs tests/python/
```

Comment:
`-b`  means "Buffer stdout and stderr during tests", seems arbitrary? :slightly_smiling_face:

Reply:
I am not sure why it's described with the word "buffer", but it actually removes all the logging messages. They are not that bad for createrepo_c, but for librepo it's really long.

Reply:
Okay. It's likely because it has to store the stdout/stderr in a buffer until a test finishes and then print it if the test failed or discard if the test passed (docs for reference: https://docs.python.org/3/library/unittest.html#cmdoption-unittest-b)

It just strikes me weird you've added the `-b` to the readme, as it's not necessary for a minimal example :slightly_smiling_face: (but I don't mind too much)